src/assets/wise5/authoringTool/components/translatable-input/translatable-input.component.html

```diff
@@ -1,6 +1,10 @@
 <ng-content *ngIf="!showTranslationInput()"></ng-content>
 <mat-form-field *ngIf="showTranslationInput()">
-  <mat-label i18n>{{ defaultLanguageLabelRef.nativeElement.innerHTML }}</mat-label>
+  <mat-label i18n
```

Comment:
I think you can remove the i18n attribute here because I don't think this mat-label needs to be translatable.

Reply:
Thanks for the catch.